README.md

```diff
@@ -11,6 +11,13 @@ this data see [Google Genomics Public Data](https://cloud.google.com/genomics/da
 1. Click on **"Compose Query"**.
 1. Copy and paste the following query into the dialog box and click on **"Run Query"**:
 ```
+# Count the number of records we have in the dataset and the total number of
+# calls nested within those records.
+# 
+# The source data for table genomics-public-data:platinum_genomes.variants
+# was gVCF so a record can be a particular variant or a non-variant segment.
+# https://sites.google.com/site/gvcftools/home/about-gvcf
+#
 SELECT
   reference_name,
   COUNT(reference_name) AS num_records,
```

Comment:
Fair point. One of those peculiar things that non-variants go into the "variants" table.

Would it be worth noting in the comment then that the query will "Count the number of records (variant and reference segments) that we have in the dataset..."?

Otherwise LGTM.